lib/money/accounting_money_parser.rb

```diff
@@ -1,7 +1,7 @@
 class AccountingMoneyParser < MoneyParser
   private
-  def extract_money(input)
+  def extract_money(input, currency = nil)
```

Comment:
Should this argument be `currency_iso` to line up with the method in `MoneyParser`?

Reply:
modified `currency_iso` since this also works with a currency object